2022-05-19 11:56:12.0780|INFO|Info|User Iserial:1302
2022-05-19 11:56:12.1020|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Detail":{"InnerException":null,"Message":"Object reference not set to an instance of an object.","StackTrace":"   at CCWFM.Web.Service.Operations.GlOperations.GlService.GetTblChequeTransactionDetailNotLinked(String vendor, String company) in D:\\CCWFM\\CCWFM.Web\\Service\\Operations\\GlOperations\\ClosingAdvanceVendorPayment.cs:line 20\r\n   at SyncInvokeGetTblChequeTransactionDetailNotLinked(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.NullReferenceException"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"Object reference not set to an instance of an object.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"Object reference not set to an instance of an object."}]},"Data":{},"InnerException":null,"StackTrace":"   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)\r\n   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.GlService.GlServiceClient.GlServiceClientChannel.EndGetTblChequeTransactionDetailNotLinked(IAsyncResult result)\r\n   at CCWFM.GlService.GlServiceClient.CCWFM.GlService.GlService.EndGetTblChequeTransactionDetailNotLinked(IAsyncResult result)\r\n   at CCWFM.GlService.GlServiceClient.OnEndGetTblChequeTransactionDetailNotLinked(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.GlService.GetTblChequeTransactionDetailNotLinkedCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.Gl.ClosingAdvanceVendorPaymentViewModel.<.ctor>b__0_0(Object s, GetTblChequeTransactionDetailNotLinkedCompletedEventArgs sv)\r\n   at CCWFM.GlService.GlServiceClient.OnGetTblChequeTransactionDetailNotLinkedCompleted(Object state)"}
